README.md

```diff
@@ -39,7 +39,54 @@ proxy is required that runs in front of Kibana.
 The deployer enables the user to generate all of the necessary
 key/certs/secrets and deploy all of the components in concert.
 
-## Defining local builds
+#### Curator
```

Comment:
It's not obvious to me what happens in the default scenario.  The case where there is an empty hash for INDEX_MGMT.  The way it reads is that I will need to add an entry for EVERY project which is not what we want.  I would add  a section here that makes it more obvious that doing nothing still gets you deleted indexes.  Also, are project names evaluated based on a wild card since the actual indexes are a combination of a date and project?